core/controlplane/config/templates/stack-template.json

```diff
@@ -158,6 +158,57 @@
         }
       }
     },
+    "SecurityGroupWorker": {
```

Comment:
This security group needed returning to the control plane.  We can remove it again in later releases but without it in the updated control plane stack we can an error about it being in use by the nodepools

Reply:
Makes sense. Fine with reviving this then.

// Wish we had something like reference counting to keep AWS resources only while they're used 😆